bookshop/db/schema.cds

```diff
@@ -11,6 +11,7 @@ entity Books : managed {
   price    : Price;
   currency : Currency;
   image    : LargeBinary @Core.MediaType: 'image/png';
+  contents : Composition of many Contents on contents.book = $self;
```

Comment:
We should not pollute our central getting-started sample, but move that to the fiori / bookstore project.